### ISSUE TITLE
No copy options for bookmarks

### ISSUE THREAD

**Issue by AkalUstat** [ShabadOS/desktop#500]

**Comment by bhajneet** [comment]
Forgot to assign yourself @saihaj 😂

**Comment by bhajneet** [comment]
This would also close #497

**Comment by saihaj** [comment]
can you give a specific example?

**Comment by saihaj** [comment]
Depends on #495

**Comment by saihaj** [comment]
Putting On Hold since #495 is needed to be implemented first.
